scripts/stablex/add_token_list.js

```diff
@@ -0,0 +1,22 @@
+const { addTokens } = require("../script_utilities.js")
+const fetch = require("node-fetch")
+
+module.exports = async function (callback) {
+  try {
+    //TODO(fleupold) use master branch here
```

Comment:
This should be addressed before merging. Dependent on merging https://github.com/gnosis/dex-react/pull/177

Reply:
@josojo  - since you are working on this at the moment. Is this still an issue?

Reply:
Now, I managed to merge it on dex-react :)